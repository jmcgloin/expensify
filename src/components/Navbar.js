import React from "react";
import { NavLink } from "react-router-dom";

const Navbar = () => (
	<nav>
		<NavLink to="/" activeClassName="is-active" exact={ true }>Home</NavLink>
		<NavLink to="/create" activeClassName="is-active">Add</NavLink>
		<NavLink to="/help" activeClassName="is-active">Help</NavLink>
		<NavLink to="/dne" activeClassName="is-active">DNE</NavLink>
	</nav>
);

export default Navbar;