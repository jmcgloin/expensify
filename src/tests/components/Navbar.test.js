import React from "react";
import { shallow } from "enzyme";
import Navbar from "../../components/Navbar";

test('should render navbar element correctly', () => {
	const wrapper = shallow(<Navbar />);
	expect(wrapper).toMatchSnapshot();
});