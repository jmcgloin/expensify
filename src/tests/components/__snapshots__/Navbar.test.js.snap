// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render navbar element correctly 1`] = `
<nav>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Add
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/dne"
  >
    DNE
  </NavLink>
</nav>
`;
